QuarantineHanabi.iml

```diff
@@ -29,13 +29,13 @@
     <orderEntry type="library" name="popper" level="application" />
   </component>
   <component name="PyDocumentationSettings">
-    <option name="format" value="GOOGLE" />
```

Comment:
I think this keeps changing because we're running different versions of IntelliJ.